backstop/tests/header.html

```diff
@@ -172,6 +172,29 @@ <h2>Default</h2>
                   aria-hidden="true"
                 ></svg-caret-down-small>
               </button>
+
+              <svg-chevron-right
+                class="iui-chevron"
+                aria-hidden="true"
+              ></svg-chevron-right>
+
+              <!-- Version Button -->
+              <button
+                class="iui-button iui-borderless iui-active iui-header-button"
+                type="button"
+                aria-label="Model picker"
+                aria-expanded="false"
+                aria-current="location"
+              >
+                <svg-model
+                  class="iui-button-icon iui-header-button-icon"
+                  aria-hidden="true"
+                ></svg-model>
+                <span class="iui-button-label">
+                  <div>Version Gama</div>
```

Comment:
```suggestion
                  <div>Version Charlie</div>
```
You can use Charlie if you want to keep the NATO phonetic alphabet ordered correctly.

Reply:
I thought this was the greek alphabet, hence gamma. http://stars.astro.illinois.edu/sow/greek.html

Just misspelled a little bit.
```suggestion
                  <div>Version Gamma</div>
```

Reply:
Yup, good catch. Did the fix.